### PR TITLE
Feature/onboardingmanager

### PR DESCRIPTION
**Pull request by dephillipsmichael**
This is the code to use the OnboardingManager in MPower.  It relies on pointing at a different branch for ResearchStack and the BridgeSDK (Android-SDK).   https://github.com/Sage-Bionetworks/BridgeAndroidSDK/pull/15 and https://github.com/ResearchStack/ResearchStack/pull/320

### CODE REVIEW

File: MPower/app/src/main/java/org/sagebionetworks/bridge/researchstack/MPowerOnboardingManager.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+/**
+ * Created by TheMDP on 1/12/17.
+ *
+ * This class is overridden to demonstrate to the developer how to convert
+ * custom JSON SurveyItem objects in "onboarding.json" into CustomSteps used
+ * in the onboarding process
+ */
+
+public class MPowerOnboardingManager extends OnboardingManager {
```
Comment (dephillipsmichael) [inline MPower/app/src/main/java/org/sagebionetworks/bridge/researchstack/MPowerOnboardingManager.java]: @EricSiegNW this class should answer your question about implementing your own SurveyItems/Steps/Tasks/etc.

File: MPower/app/src/main/java/org/sagebase/mpower/OnboardingActivity.java [hunk 2]
```diff
@@ -10,16 +10,17 @@
...
 import org.researchstack.skin.model.StudyOverviewModel;
+import org.researchstack.skin.ui.OverviewActivity;
 import org.researchstack.skin.ui.adapter.OnboardingPagerAdapter;
 
 import java.util.ArrayList;
 import java.util.List;
 
 
-public class OnboardingActivity extends org.researchstack.skin.ui.OnboardingActivity {
+public class OnboardingActivity extends OverviewActivity {
```
Comment (rianhouston) [inline MPower/app/src/main/java/org/sagebase/mpower/OnboardingActivity.java]: Should this keep the name OverviewActivity?  Does it do more than the overview?

**Comment by rianhouston** [comment]
Looks pretty good to me.  Just a minor comment.

File: MPower/app/src/main/java/org/sagebase/mpower/OnboardingActivity.java (hunk 2 above)
Comment (dephillipsmichael) [inline MPower/app/src/main/java/org/sagebase/mpower/OnboardingActivity.java]: Good catch, it should probably be renamed MPowerOverviewActivity

**Pull request merged**